scheduler/server/coordinator_test.go

```diff
@@ -367,104 +367,104 @@ func (s *testCoordinatorSuite) TestPeerState(c *C) {
-	c.Assert(err, IsNil)
-	tc := newTestCluster(opt)
-	hbStreams, cleanup := getHeartBeatStreams(s.ctx, c, tc)
-	defer cleanup()
-	defer hbStreams.Close()
-
-	co := newCoordinator(s.ctx, tc.RaftCluster, hbStreams)
-
-	c.Assert(tc.addLeaderStore(1, 5), IsNil)
-	c.Assert(tc.addLeaderStore(2, 2), IsNil)
-	c.Assert(tc.addLeaderStore(3, 0), IsNil)
-	c.Assert(tc.addLeaderStore(4, 0), IsNil)
-	c.Assert(tc.LoadRegion(1, 1, 2, 3), IsNil)
-	c.Assert(tc.LoadRegion(2, 1, 2, 3), IsNil)
-	c.Assert(tc.LoadRegion(3, 1, 2, 3), IsNil)
-	c.Assert(tc.LoadRegion(4, 1, 2, 3), IsNil)
-	c.Assert(tc.LoadRegion(5, 1, 2, 3), IsNil)
-	c.Assert(tc.LoadRegion(6, 2, 1, 4), IsNil)
-	c.Assert(tc.LoadRegion(7, 2, 1, 4), IsNil)
-	c.Assert(co.shouldRun(), IsFalse)
-	c.Assert(tc.core.Regions.GetStoreRegionCount(4), Equals, 2)
-
-	tbl := []struct {
-		regionID  uint64
-		shouldRun bool
-	}{
-		{1, false},
-		{2, false},
-		{3, false},
-		{4, false},
-		{5, false},
-		// store4 needs collect two region
-		{6, false},
-		{7, true},
-	}
-
-	for _, t := range tbl {
-		r := tc.GetRegion(t.regionID)
-		nr := r.Clone(core.WithLeader(r.GetPeers()[0]))
-		c.Assert(tc.processRegionHeartbeat(nr), IsNil)
-		c.Assert(co.shouldRun(), Equals, t.shouldRun)
-	}
-	nr := &metapb.Region{Id: 6, Peers: []*metapb.Peer{}}
-	newRegion := core.NewRegionInfo(nr, nil)
-	c.Assert(tc.processRegionHeartbeat(newRegion), NotNil)
-	c.Assert(co.cluster.prepareChecker.sum, Equals, 7)
-
-}
-func (s *testCoordinatorSuite) TestShouldRunWithNonLeaderRegions(c *C) {
-	_, opt, err := newTestScheduleConfig()
-	c.Assert(err, IsNil)
-	tc := newTestCluster(opt)
-	hbStreams, cleanup := getHeartBeatStreams(s.ctx, c, tc)
-	defer cleanup()
-	defer hbStreams.Close()
-
-	co := newCoordinator(s.ctx, tc.RaftCluster, hbStreams)
-
-	c.Assert(tc.addLeaderStore(1, 10), IsNil)
-	c.Assert(tc.addLeaderStore(2, 0), IsNil)
-	c.Assert(tc.addLeaderStore(3, 0), IsNil)
-	for i := 0; i < 10; i++ {
-		c.Assert(tc.LoadRegion(uint64(i+1), 1, 2, 3), IsNil)
-	}
-	c.Assert(co.shouldRun(), IsFalse)
-	c.Assert(tc.core.Regions.GetStoreRegionCount(1), Equals, 10)
-
-	tbl := []struct {
-		regionID  uint64
-		shouldRun bool
-	}{
-		{1, false},
-		{2, false},
-		{3, false},
-		{4, false},
-		{5, false},
-		{6, false},
-		{7, false},
-		{8, true},
-	}
-
-	for _, t := range tbl {
-		r := tc.GetRegion(t.regionID)
-		nr := r.Clone(core.WithLeader(r.GetPeers()[0]))
-		c.Assert(tc.processRegionHeartbeat(nr), IsNil)
-		c.Assert(co.shouldRun(), Equals, t.shouldRun)
-	}
-	nr := &metapb.Region{Id: 8, Peers: []*metapb.Peer{}}
-	newRegion := core.NewRegionInfo(nr, nil)
-	c.Assert(tc.processRegionHeartbeat(newRegion), NotNil)
-	c.Assert(co.cluster.prepareChecker.sum, Equals, 8)
-
-	// Now, after server is prepared, there exist some regions with no leader.
-	c.Assert(tc.GetRegion(9).GetLeader().GetStoreId(), Equals, uint64(0))
-	c.Assert(tc.GetRegion(10).GetLeader().GetStoreId(), Equals, uint64(0))
-}
+//func (s *testCoordinatorSuite) TestShouldRun(c *C) {
```

Comment:
why comment out

Reply:
Oops sorry. They should be deleted, as `shouldRun` is deleted now.